package.json

```diff
@@ -2,8 +2,10 @@
   "name": "@tl-its-umich-edu/react-ga-onetrust-consent",
```

Comment:
When I do the `npm install` I see 1 moderate severity vulnerability. When you run the `npm install report` this issue is easily fixable with `npm audit fix`